pkg/controller/pxc/backup.go

```diff
@@ -36,30 +39,28 @@ func (r *ReconcilePerconaXtraDBCluster) reconcileBackups(cr *api.PerconaXtraDBCl
 	backupNamePrefix := backupJobClusterPrefix(cr.Name)
 
 	if cr.Spec.Backup != nil {
-
 		if cr.Status.Status == api.AppStateReady && cr.Spec.Backup.PITR.Enabled && !cr.Spec.Pause {
 			binlogCollector, err := deployment.GetBinlogCollectorDeployment(cr)
 			if err != nil {
 				return errors.Errorf("get binlog collector deployment for cluster '%s': %v", cr.Name, err)
 			}
-			binlogCollectorName := deployment.GetBinlogCollectorDeploymentName(cr)
+
 			currentCollector := appsv1.Deployment{}
-			err = r.client.Get(context.TODO(), types.NamespacedName{Name: binlogCollectorName, Namespace: cr.Namespace}, &currentCollector)
+			err = r.client.Get(context.TODO(), types.NamespacedName{Name: binlogCollector.Name, Namespace: cr.Namespace}, &currentCollector)
 			if err != nil && k8serrors.IsNotFound(err) {
```

Comment:
Just for my understanding why we use cr.Namespace not binlogCollector.Namespace? (I understand that the value the same but still)

Reply:
no reason, you're right:
https://github.com/percona/percona-xtradb-cluster-operator/pull/1245/commits/65924851d614b363f27743d9cce656f3210409b5